### ISSUE TITLE
Text encoding for TWKB?

### ISSUE THREAD

**Issue by dr-jts** [TWKB/Specification#24]
Is there a specified text encoding for TWKB?  

Possibilities for text encoding include Hex and Base64.  There may be other better ones. It would be nice if the spec mandated a standard encoding.  This would allow more standard use of TWB for things like encoding geometry inside JSON.

**Comment by nicklasaven** [comment]
Yes, why not. I guess specifying things makes it more usable. I think base64 sounds good.

Do you make a PR?

**Comment by dr-jts** [comment]
I will make a PR when I have time.
